test/DynamoCoreTests/UnitTestBase.cs

```diff
@@ -65,6 +66,19 @@ public virtual void Setup()
         {
             SetupDirectories();
             DSOffice.ExcelInterop.ShowOnStartup = false;
+
+            if (assemblyHelper == null)
+            {
+                var assemblyPath = Assembly.GetExecutingAssembly().Location;
+                var moduleRootFolder = Path.GetDirectoryName(assemblyPath);
+                var resolutionPaths = new[]
+                {
+                    // These tests need "CoreNodeModels.dll" under "nodes" folder.
```

Comment:
it may be good to run this PR on master-15 self serve to see if this causes any problems.

Reply:
will run when all PRs are merged into this one